scripts/utils/climate_assessment/submit_climate_assessment.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#SBATCH --qos=priority
+#SBATCH --time=06:00:00
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=12
+#SBATCH --mem=60000
+#SBATCH --job-name=test_remind_2
+#SBATCH --output=PYTHONLOG-%x.%j.out
+# Replace this with the resulting xls of output.R -> export -> xlsx_IIASA -> AR6
+filename="../../../output/export/REMIND_gabrielAR6SHAPE_2023-05-17_05.12.52.xlsx"
+python source_climate_assessment.py $filename
```

Comment:
I agree, but I think passing command line arguments through sbatch is not very straightforward. Since the plan is to replace this with a output.R workflow, maybe we merge this now and I patch it if it's taking too long to do the permant one.